beacon-chain/blockchain/receive_attestation.go

```diff
@@ -190,9 +190,13 @@ func (s *Service) processAttestations(ctx context.Context, disparity time.Durati
 		}
 
 		if err := s.receiveAttestationNoPubsub(ctx, a, disparity); err != nil {
+			ci, err := a.GetCommitteeIndex()
```

Comment:
wonder if we should do it this way or not

Reply:
This code deals with forkchoice attestations which includes attestations coming from blocks. And these attestations can have more than one bit set. You should write something like https://github.com/prysmaticlabs/prysm/blob/fad92472d87a47f3ad28abbf32df8391ce0345f8/beacon-chain/rpc/core/validator.go#L419-L434

Reply:
hmm I think this would ultimately do the same thing, the committee index returned will be based on its underlying type